src/main/java/org/testeditor/fixture/host/HostDriverFixture.java

```diff
@@ -81,13 +85,27 @@ protected HostDriverFixture(Connection connection) {
      *            The hostname to be connected to.
      * @param port
      *            The port of the host to be connected to.
+     * @param offsetRow
+     *            The offsetRow for the screen origin. Default for the x3270
+     *            driver is that the row and column are zero-origin (0;0), that
+     *            means the begin point is at row 0 and column 0. The offsetRow
+     *            will rearrange the startpoint around the value for the row .
+     * @param offsetColumn
+     *            The offsetColumn for the screen origin. Default for the x3270
+     *            driver is that the row and column are zero-origin (0;0), that
+     *            means the begin point is at row 0 and column 0. The
+     *            offsetColumn will rearrange the startpoint around the value
+     *            for the column.
      */
     @FixtureMethod
-    public boolean connect(String s3270Path, String hostname, int port) {
+    public boolean connect(String s3270Path, String hostname, int port, int offsetRow, int offsetColumn) {
         logger.info("Host-Fixture connecting ...");
+        Offset offset = new Offset(-offsetRow, -offsetColumn);
```

Comment:
Why are the values negated here? And why not save the offset instance as `this.offset`?